hack/unwanted-dependencies.json

```diff
@@ -27,7 +27,6 @@
       "github.com/influxdata/influxdb1-client": "",
       "github.com/json-iterator/go": "refer to #105030",
       "github.com/miekg/dns": "no dns client/server should be required",
-      "github.com/mindprince/gonvml": "depends on nvml.h that does not appear to permit modification, redistribution",
```

Comment:
We can keep this line here so that we can avoid re-including this dependency to k/k.

Reply:
+1